common/include/pcl/common/fft/_kiss_fft_guts.h

```diff
@@ -15,7 +15,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 /* kiss_fft.h
    defines kiss_fft_scalar as either short or a float type
    and defines
-   typedef struct { kiss_fft_scalar r; kiss_fft_scalar i; }kiss_fft_cpx; */
+   using r = kiss_fft_cpx { kiss_fft_scalar; kiss_fft_scalar i; }; */
```

Comment:
This is wrong I believe

Reply:
Agreed. There's no separate namespace between structs and the remaining types in C++. This feels like a remnant from C code.

Reply:
Well, seems my Regex to replace everything clang-tidy missed, should be a bit adjusted ;).